pkg/render/fluentd.go

```diff
@@ -166,6 +169,7 @@ type FluentdConfiguration struct {
 	UsePSP bool
 	// Whether to use User provided certificate or not.
 	UseSyslogCertificate bool
+	EnterpriseCRDExists  bool
```

Comment:
If we need this flag in this config it should be based on the Variant rather than CRD existence.